pulsar-client/pom.xml

```diff
@@ -90,6 +90,12 @@
          </exclusion>
       </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
Client should depend on protobuf. If the object is indeed in protobuf, the application should already have protobuf in the classpath. I believe the scope for this dependency should be set to `provided`.

Reply:
+1